NuGet.Config

```diff
@@ -16,6 +16,6 @@
     <add key="vctools" value="https://vcppdogfooding.azurewebsites.net/nuget/" />
     <add key="myget.org roslyn tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
     <add key="myget.org roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
-    <add key="artifacts" value="../artifacts" />
+    <add key="artifacts" value="artifacts" />
```

Comment:
Where is the artifacts directory?  
in the existing build it is a subdirectory of the root of the repo.